Content.Server/Radio/EntitySystems/RadioDeviceSystem.cs

```diff
@@ -1,5 +1,6 @@
 using Content.Server.Chat.Systems;
 using Content.Server.Interaction;
+using Content.Server.Language;
 using Content.Server.Popups;
 using Content.Server.Power.Components;
 using Content.Server.Power.EntitySystems;
```

Comment:
Also need this to prevent an error around line 16 is good.
`using Content.Shared.UserInterface;`

Still a lot of errors I'm getting in radiosystem.cs and some in languagesystem.cs

Reply:
Do note that ChatSystem was edited before by the Vulpkanin PR.

Reply:
Yeah looks like it's a mess trying to run this on upstream codebase since there's a ton of little changes in other areas.